ChegeBryan/politico
# #163966060 User voting API

#### What does this PR do?

Builds votes registering endpoint.

#### Description of Task to be completed?

- [x] Add tests for the vote API
- [x] Add vote validation tests.
- [x] Create votes endpoint 

#### How should this be manually tested?
#### How should this be manually tested?
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Using
[Postman](https://www.getpostman.com/) open a request to create a party [How to create a party](https://github.com/ChegeBryan/politico/pull/45). create an office [How to create an office](https://github.com/ChegeBryan/politico/pull/50). Follow the instructions on the PR's on how to make those requests.
Once you have an office and party also register a user [how to signup user](https://github.com/ChegeBryan/politico/pull/39). To register a user as a candidate for the office created [How to register candidate](https://github.com/ChegeBryan/politico/pull/55) Once you have a candidate Open a request, add user token the authorization header,  with on this endpoint `POST /api/v2/votes`
set content type to JSON and add the following in the json body. 
```JSON
{
	"office": 1,
	"candidate": 2
}
```

#### What are the relevant pivotal tracker stories?
[#163966060](https://www.pivotaltracker.com/story/show/163966060)

#### Screenshots (if appropriate)
##### `POST /api/v2/votes`
![Screenshot from 2019-04-01 00-45-54](https://user-images.githubusercontent.com/40359451/55295596-9dcdc500-5417-11e9-90cb-3c030f6cfe98.png)
